sdk/core/core/src/az_pal.c

```diff
@@ -0,0 +1,23 @@
+#include <az_pal.h>
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+
+#include <_az_cfg.h>
+
+#ifdef _WIN32
+void az_pal_wait(int32_t milliseconds) {
```

Comment:
This can't stay like this. Each pal implementation MUST be inits own .c file. We can wait for Charlie  to get back to us.

Reply:
I know, this is temporary, until we have a solution from Charlie.